view/frontend/web/js/view/checkout/email.js

```diff
@@ -34,6 +34,10 @@ define([
       var self = this;
       console.log('Klaviyo_Reclaim - Binding to #customer-email');
       jQuery('#maincontent').delegate('#customer-email', 'change', function (event) {
+        if (!window._learnq || !window._learnq.identify) {
```

Comment:
looks good, just one comment

could we put this before the if/else in the `handleCheckout`? that way we don't get into the postUserEmail method?
Preferable something along the lines of:

```
handleCheckout: function(){
  if (!this.isKlaviyoActive()) {
    return;
  }
  ....
}

isKlaviyoActive: function(){
    if (!window._learnq || !window._learnq.identify) {
      return;
    }
    return true;
}
```

Reply:
Klaviyo js initialize asynchronously, so during initialization it might be still not available

Reply:
that's fine, still make the function isKlaviyoActive please and will ship with #54

Reply:
Added isKlaviyoActive function, but kept it inside "delegate" function, in order not to brake anything